src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -383,8 +383,8 @@ private void testGetFeedbackSessionDetailsForInstructor() throws Exception {
 
         // 2 instructors, 6 students = 8
         assertEquals(8, stats.expectedTotal);
-        // 1 instructor, 1 student, did not respond => 8-2=6
-        assertEquals(6, stats.submittedTotal);
+        // 5 student , 1 instructor and 2 Team responded => 5+1+2=8
```

Comment:
If we consider like this then it is possible that `submittedTotal > expectedTotal`

We should only consider respondents into the `submittedTotal` and not Teams.

Reply:
@tshradheya  what I exactly need to do here